test/specification/commonmark.py

```diff
@@ -34,7 +34,7 @@ def run_tests(test_entries, start=None, end=None,
 def run_test(test_entry, quiet=False):
     test_case = test_entry['markdown'].splitlines(keepends=True)
     try:
-        output = markdown(test_case).replace('&#x27;', "'")
```

Comment:
Does this mean that the CommonMark tests now pass without special handling of escaped quotes? If yes, then that's a big win!

Reply:
Yes. I would say a little win here. ;)